build.gradle

```diff
@@ -177,8 +177,8 @@ apply plugin: 'org.sonarqube'
 
 sonarqube {
     properties {
-        property "sonar.projectName", "Element-Android"
-        property "sonar.projectKey", "im.vector.app.android"
+        property "sonar.projectName", "element-android"
+        property "sonar.projectKey", "vector-im_element-android"
```

Comment:
So the Sonar URL has changed? If yes, there are 3 lines to update in the file `README.md`

Reply:
Have replaced with newly generated links - the existing ones seemed to be legacy versions